src/style.css

```diff
@@ -1 +1,61 @@
-/* Styles go here */
+.header {
+  box-shadow: 0 2px 4px 0 rgba(0, 0, 0, 0.05);
+  display: flex;
+  position: absolute;
```

Comment:
let's not use absolute positioning in this task, just remove margin from the body (margin: 0) and your header will be already at the top of the page